README.md

```diff
@@ -119,6 +124,162 @@ resource provided above.
     $ git remote add upstream https://github.com/shescoding/shescoding-dot-org.git
     ```
 
++ To complete installation, skip past the directions for Windows 10 [here](#mac,-ubuntu,-and-windows)
```

Comment:
[Go here](#mac-ubuntu-and-windows) to complete installation, skip past the directions for Windows 10.

the link was no working

Reply:
This has been fixed